src/sentry/api/endpoints/organization_user_rollback.py

```diff
@@ -40,7 +40,10 @@ def get(self, request: Request, organization: Organization) -> Response:
 
         # Validate that the organization has rollbacks enabled
         if not organization.get_option("sentry:rollback_enabled", ROLLBACK_ENABLED_DEFAULT):
-            return Response(status=status.HTTP_404_NOT_FOUND)
+            return Response(
+                status=status.HTTP_403_FORBIDDEN,
+                data={"detail": "Rollbacks are disabled for this organization."},
```

Comment:
Could you also include a `code` with a static string value that we can read on the frontend? Makes showing the error state less fragile